bot.py

```diff
@@ -52,14 +69,22 @@ def _filter_harsh(self, tweet, tag):
                     break
             if tag not in tag_list:
                 return False
-        length = len(tweet_list) - len(tag_list)
-        tagless_list = tweet_list[:length]
-
-        tweet = " ".join(tagless_list).replace("#", "")
-        Tweet.create_or_get(text=tweet)
-
-        return {'text': tweet, 'tags': tag_list}
-
+        # length = len(tweet_list) - len(tag_list)
+        # tagless_list = tweet_list[:length]
+        # tweet.tag_list = tag_list
+        # tweet.filtered_text = " ".join(tagless_list)  # .replace("#", "")
+        user = model.User.create_or_get(screen_name=tweet.user.screen_name,
```

Comment:
peewee.create_or_get returns a tuple.  The object and a boolean as to whether it was created or not (just retrieved)
i.e.:
user, created = User.create_or_get(username=username)